whitepaper/whitepaper.md

```diff
@@ -124,13 +134,235 @@ Blockchain technologies are growing in usage, but fragmentation is a big problem
+
+Burning of assets can be implemented as follows: 
+
+* Assets are sent to the address of the coinbase/generation transaction<sup>[h](#22-footnotes-introduction)</sup> in the genesis block. A coinbase/generation transaction is in every block of blockchains that rely on mining. It is the address where the reward for mining the block is sent to. Hence, this will burn the tokens/coins in the address of the miner that mined the genesis block. In many blockchain platforms, it is proven that nobody has the private key to this special address. 
+
+* Tokens/Coins are subtracted from the user account as well as optionally from the total token/coin supply value.
+
+
+
+
+
+## 2.2 Footnotes (Introduction)
+a: There is an alternative approach for an outside entity A to verify the state of a connected blockchain if this connected blockchain uses Merkle Trees to store its blockchain state. An outside entity A can store the Merkle Tree roots from the headers of committed blocks of a connected blockchain locally to verify any state claims about the connected blockchain. Any untrusted entity can then provide a state of the connected blockchain, such as a specific account balance on the connected blockchain, because the outside entity A can act as a lightweight client and use concepts like simple payment verification (SPV) to verify that the state claim provided by the untrusted entity is valid. SPV can be done without checking the entire blockchain history. Polkadot uses this approach in its Relay Chain and the BTCRelay on the Ethereum blockchain uses this approach as well. Private blockchains do not always keep track of their state through Merkel trees and signatures produced by nodes participating in such private blockchains are rarely understood by outside parties not participating in the network. For that reason, the design principle of `Cactus` is to rely on the canonical validator node signatures for verifying proofs of blockchain states. Since `Cactus` should be able to incorporate any type of blockchain in the future, `Cactus` can not use the approach based on Merkle Trees. 
+
```

Comment:
I did not understand from "Private blockchains" up to "merkle trees". Why can't Cactus utilize a merkle tree hash as a proof of block validity? I think this could be an easy way to prove that a transaction we issued to Bitcoin or Ethereum, for example, was included into a valid block.

In fact, Cactus' validators could in theory build a merkle tree from a Fabric network, sign the hash of the merkle tree root, and deliver it as a summary/proof of the blockchain state.  

However, merkle trees are not used in Fabric, for example, because Fabric relies on endorsers' signatures to validate transactions - Fabric was not meant to be easily verifiable from the outside.

Reply:
Cactus cannot rely natively on Merkle tree proofs since not all blockchains support it natively. Of course, it is possible to reproduce the complete state of a blockchain in a Merkle tree but it might cost you a lot of computational resources or storage if you only want to get a proof quite infrequently. It would kind of duplicate the world state of the blockchain but if that is not of your concern then yes a cactus validator node can be implemented to produce proofs of state of a blockchain by generating a Merkle tree.

Reply:
I understand now what you meant. How would it duplicate the world state?